src/styles/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: transform 0.3s ease;
+  color: $black;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    height: 166px;
+    background-position: 19px 32px;
+  }
+
+  &__apple {
+    margin: $margin;
```

Comment:
The variable `$margin` is used here, but it's not defined in this file. Ensure that this variable is defined elsewhere in your stylesheets or replace it with a fixed value.